src/lib389/cli/dscontainer

```diff
@@ -425,6 +427,19 @@ def begin_healthcheck(ds_proc, log_exception):
     return (True, False)
 
 
+def stop():
+    pid=int(open(PID_FILE).read())
+    parent = psutil.Process(pid)
+    procs = parent.children(recursive=True)
+    procs.append(parent)
+    for p in procs:
+        p.terminate()
+
+    gone, alive = psutil.wait_procs(procs, timeout=3)
```

Comment:
I think the timeout should be a bit longer than 3 seconds here.

Reply:
Good idea.  Will fix.